app/classes/set_mappings_used.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class SetMappingsUsed
+  include HTTParty
+  base_uri "https://launchpad-accelerator.herokuapp.com"
+
+  attr_reader :mappings_used
+
+  def initialize(mappings_used)
+    @mappings_used = mappings_used
+  end
+
+  def call
+    options = {
+      body: {
+        app: {
+          license_key: ENV["LAUNCHPAD_LICENSE_KEY"],
+          mappings_used: mappings_used,
```

Comment:
Kind of a larger item to discuss but this could be mutated to set a count lower than what is actually used by editing this code. To confirm, this is the code that is installed by a customer of the add-on? Seems like we don't have a guard against lowering the mappings used https://github.com/LaunchPadLab/launchpad_accelerator_api/blob/master/app/controllers/apps_controller.rb#L69